CODE_STYLE.md

```diff
@@ -25,7 +25,7 @@ code style should not be changed and must be followed.
 
 - Typescript
 
-- React.js 
+- React.js
 
 - CSS module
 
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project